jena-core/src/main/java/org/apache/jena/rdf/model/Model.java

```diff
@@ -1051,4 +1051,12 @@ Remove all the statements matching (s, p, o) from this model.
         Answer true iff .close() has been called on this Model.
     */
     public boolean isClosed();
+
+    // Override return type for methods inherited from PrefixMapping
```

Comment:
We should use `@Override` for explicitness on these changes.

Reply:
Done.